backend/main.py

```
@@ -120,26 +160,123 @@ def login(form_data: OAuth2PasswordRequestForm = Depends(), db: Session = Depend
+            "date": r.created_at
+        } for r in records
+    ]}
 
 
 # =========================================================
-# 4. STREAMING ENDPOINT (NDJSON + DB)
+# 6. STREAMING ENDPOINT (NDJSON + DB + PATIENT + IMAGE SAVE)
 # =========================================================
 @app.post("/analyze-glaucoma-stream")
-async def analyze_glaucoma_stream(file: UploadFile = File(...), current_user: User = Depends(get_current_user), db: Session = Depends(get_db)):
+async def analyze_glaucoma_stream(
```

Ruff (0.15.20)

[warning] Too many statements (66 > 50)

(PLR0915)